### DISCUSSION TITLE
Pull ltree data directly from PostgreSQL

### DISCUSSION THREAD

**Discussion by streamtree** [jinagamvasubabu/LtreeVisualizer#2] [Ideas]
Great start! It would be nice to be able to specify an alternative config file that would allow the ltree data to be pulled directly from the database rather than having to go through an interim JSON file.

**Comment by jinagamvasubabu** [comment]
Thanks @streamtree for this suggestion. Alternative config file you mean, config file contains postgres connection urls and table names and the column names to fetch the ltree data ?

correct me if i don't understand your thought well

**Reply by streamtree** [reply]
Exactly.

**Reply by jinagamvasubabu** [reply]
Done @streamtree. 

Provided a new way to fetch the ltree data to be pulled directly from the database instead of interim json file

more info - https://github.com/jinagamvasubabu/ltreevisualizer#db-way-directly-fetch-the-data-from-your-postgres-db
